shogun-boot/src/main/resources/templates/index.html

```diff
@@ -132,81 +136,90 @@ <h3 class="pb-2 pt-4">Applications</h3>
 
   <script th:inline="javascript">
     window.onload = async () => {
-      const keycloakHost = [[${@environment.getProperty('KEYCLOAK_HOST')}]];
-      const keycloak = new Keycloak({
-        url: `https://${keycloakHost}/auth`,
-        realm: 'SHOGun',
-        clientId: 'shogun-client'
+      const keycloakHost = [[${@environment.getProperty('KEYCLOAK_HOST')
+    }]];
+    const keycloak = new Keycloak({
+      url: `https://${keycloakHost}/auth`,
+      realm: 'SHOGun',
+      clientId: 'shogun-client'
+    });
+    try {
+      const authenticated = await keycloak.init({
+        onLoad: 'check-sso'
       });
-      try {
-        const authenticated = await keycloak.init({
-          onLoad: 'check-sso'
-        });
-      } catch (error) {
-        console.error(error);
-        return;
-      }
+    } catch (error) {
+      console.error(error);
+      return;
+    }
 
-      // Register login/logout actions
-      document.querySelector('#login-btn').addEventListener('click', () => {
-        keycloak.login();
-      });
-      document.querySelector('#logout-btn').addEventListener('click', () => {
-        keycloak.logout();
-      });
-      if (keycloak.authenticated) {
-        document.querySelector('#logout-btn').hidden = false;
-        document.querySelector('#login-btn').hidden = true;
-      }
+    // Register login/logout actions
+    document.querySelector('#login-btn').addEventListener('click', () => {
+      keycloak.login();
+    });
+    document.querySelector('#logout-btn').addEventListener('click', () => {
+      keycloak.logout();
+    });
+    if (keycloak.authenticated) {
+      document.querySelector('#logout-btn').hidden = false;
+      document.querySelector('#login-btn').hidden = true;
+    }
 
-      const applications = await getApplications(keycloak.token);
-      const appInfos = applications.map(app => ({
-        id: app.id,
-        name: app.name,
-        description: app.clientConfig && app.clientConfig.description
-          ? app.clientConfig.description
-          : ''
-      }));
-      const appsEl = document.querySelector('.apps');
+    const applications = await getApplications(keycloak.token);
+    const appInfos = applications.map(app => ({
+      id: app.id,
+      name: app.name,
+      logo: app.clientConfig && app.clientConfig.theme
```

Comment:
What happens if clientConfig and clientConfig.theme is defined but no logopath is present?